src/vm.c

```diff
@@ -27,9 +27,6 @@
 extern "C" {
 #endif
 void abort(void);
-#if defined(__cplusplus)
```

Comment:
i don't think this is quite right and I'm not sure if this works if `MRB_NO_STDIO` is not defined.

These are all of the flags I'm passing to clang++:

```
            .arg("-DMRB_ARY_NO_EMBED")
            .arg("-DMRB_GC_TURN_OFF_GENERATIONAL")
            .arg("-DMRB_INT64")
            .arg("-DMRB_NO_BOXING")
            .arg("-DMRB_NO_PRESYM")
            .arg("-DMRB_NO_STDIO")
            .arg("-DMRB_USE_CXX_EXCEPTION")
            .arg("-DMRB_UTF8_STRING");
```